### PR TITLE
Updated copyright info for README

### LINKED ISSUE #1251: Update copyright information on README
Per feedback from legal: Update copyright info on README with: 

>>> go-ipfs:

The go-ipfs project is dual-licensed under Apache 2.0 and MIT terms, as explained in the [Permissive License Stack](https://protocol.ai/blog/announcing-the-permissive-license-stack/):

Apache License, Version 2.0, ([LICENSE-APACHE](https://github.com/ipfs/go-ipfs/blob/master/LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
MIT license ([LICENSE-MIT](https://github.com/ipfs/go-ipfs/blob/master/LICENSE-MIT) or http://opensource.org/licenses/MIT)

### PR DESCRIPTION
**Pull request by dawnkelly09**
Per feedback from legal Fixes #1251

### CODE REVIEW

**Comment by dawnkelly09** [comment]
This is waiting on final clarification from Legal team. Do not merge yet! Thanks!

**Comment by johnnymatthews** [comment]
Converted this PR to draft, just until it's ready for review.

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+We would **love ❤️ your help** to improve existing items or make new ones even better! <!--[We also have bounties available](https://github.com/ipfs/devgrants/projects/1)!-->
+
+### Issues
+
+If you find something wrong within this repository, please raise an [issue here →](https://github.com/filecoin-project/filecoin-docs/issues). Unless the issue is urgent, updates will be batch-merged into `main` on Tuesdays or Thursdays.
+
+<!--### Bounties
+
+You can earn the undying love of the Filecoin community _and_ make some money by closing an issue with the [`bounty` tag](https://github.com/ipfs/ipfs-docs/issues?q=is%3Aopen+is%3Aissue+label%3Abounty)! Submissions must be production-ready and meet all the specifications listed on the issue page. Check out the [current list of open bounties →](https://github.com/ipfs/devgrants/projects/1) -->
+
```
Comment (johnnymatthews) [inline readme.md:22]:
```suggestion
```

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+1. Close the local server with `CTRL` + `c`.
+1. To restart the local server, run `npm start` from within the `filecoin-docs` folder.
+1. To publish the site run: ```npm run build``` The project will be built and saved to /public.
+
+## Core members
+
+<!--TO-DO: keep these same key members or update for more Filecoin specific people?-->
+
+- [@johnnymatthews](https://github.com/johnnymatthews): Project leadership, organization, and primary contact
+- [@cwaring](https://github.com/cwaring): Development support
```
Comment (johnnymatthews) [inline readme.md:66]:
```suggestion
```

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+
+
```
Comment (johnnymatthews) [inline readme.md:2]:
```suggestion
# Filecoin Docs
```

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Filecoin Docs
```
Comment (johnnymatthews) [inline readme.md:1]:
```suggestion
# Filecoin Docs

```

**Comment by mishmosh** [comment]
I think this PR is now outdated (since it adds the README.md file afresh, which was completed in #1249. Should merge `main` into this branch?

**Comment by johnnymatthews** [comment]
Just merged `main` into this branch, so everything's all caught up now.

**Comment by mishmosh** [comment]
@dawnkelly09 It's because you created this branch from `main` before #1249 was merged, so to github it looks like you're effectively PRing the entire new readme.md file twice. 

More info here: https://docs.gpcrdb.org/git_workflow.html

**Comment by dawnkelly09** [comment]
Please just confirm if we are all happy with this so it can be merged. Or if it is better to just close it at this point. Thank you!

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+1. Open [localhost:1313](http://localhost:1313/) in your browser.
+1. Close the local server with `CTRL` + `c`.
+1. To restart the local server, run `npm start` from within the `filecoin-docs` folder.
+1. To publish the site run: ```npm run build``` The project will be built and saved to /public.
+
+## License
+
+The Filecoin documentation project is dual-licensed under Apache 2.0 and MIT terms, as explained in the Permissive License Stack:
+
+Apache License, Version 2.0, (LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0)
```
[the remark this answers is not in the document]
Comment (mishmosh) [reply readme.md:62]: This repeats info already contained in line 60. `LICENSE-APACHE` would usually link to a new file containing a copy of the license, but for simplicity's sake let's not create new license files solely for the docs project, we can link to the published versions.

**Review by mishmosh — CHANGES_REQUESTED** [review]
Suggestions made for lines 60-63.

**Pull request merged**